support-frontend/assets/helpers/forms/paymentIntegrations/payPalRecurringCheckout.ts

```diff
@@ -244,14 +251,17 @@ function getPayPalOptions(
 	onPayPalCheckoutCompleted: (arg0: PayPalCheckoutDetails) => void,
 	canOpen: () => boolean,
 	onClick: () => void,
-	formClassName: string,
+	_formClassName: string,
```

Comment:
If `_formClassName` isn't used within `getPayPalOptions` then can we stop passing it in?